scripting/retakes_pistolallocator.sp

```diff
@@ -20,6 +20,8 @@ char g_PistolChoices[][][] = {
     { "weapon_tec9", "Tec-9" },
     { "weapon_fiveseven", "Five-Seven" },
     { "weapon_deagle", "Deagle" },
+    // Random Pistol
```

Comment:
Can you adjust to comment to:

// Random pistol, must be last entry in the array.

Reply:
Changed